src/NuGet.Indexing/ILoader.cs

```diff
@@ -16,5 +16,7 @@ public interface ILoader
         /// <param name="config">Configuration to use.</param>
         /// <returns>Returns true if the data has changed.</returns>
         bool Reload(IndexingConfiguration config);
+
+        DateTime? GetLastUpdateTime(string name);
```

Comment:
From my understanding, there are few key timestamps here.
1. Last time the aux reload was attempted
2. last time the aux reload succeeded
3. Timestamp on the currently loaded aux files.

Since there is some delay on when the aux files are recorded and reloaded, there can be cases where the last reload changes, but the timestamp on the actual file has not changed. This is due to all our updates/reloads being on different cadences (ie index reload callback happens every X minutes, while aux data timeout is another every X minute, and aux data update [from jobs] is every X hours).

Since this is an endpoint view, we want to see when and if there is a problem here. Last reload time of this would tell us if the service is working, whereas the last timestamp on the file could potentially tell us if updating the files is failing. Ideally, we would have separate monitoring of the auxiliary data files, at which point the last updated timestamp information here could be deprecated.